code/dqc/symfilegrowth.q

```diff
@@ -0,0 +1,19 @@
+\d .dqc
+
+/- Take the average symfile count from the past n days, then check that todays
+/- sym file count hasn't grown more than pct%.
+symfilegrowth:{[ndays;pct]
+  .lg.o[`dqe;"Checking sym file has not grown more than ",string[pct],"%."];
```

Comment:
Last nitpick, change is of this log message and the other on line 17 from `dqe` to something like `symfilegrowth`